tests/playwright/src/ai-lab-extension.spec.ts

```diff
@@ -143,6 +157,23 @@ test.describe.serial(`AI Lab extension installation and verification`, () => {
           .toBeTruthy();
       });
 
+      test(`Expect ${modelName} is in available models list`, async ({ request }) => {
```

Comment:
Making it more clear that you are testing api access, this is not clear from the test case name.